src/components/SearchBar.js

```diff
@@ -367,56 +412,81 @@ class SearchBar extends React.Component<SearchBarDefaultProps, SearchBarProps, S
           {advancedMenuItem}
         </Dropdown.Menu>
       </Dropdown>
-    ) : '';
+    ) : (
+      ''
+    );
 
     let placeholder = this.props.placeholder;
     if (this.props.allowLanguageSelect && language === 'advanced') {
       placeholder = this.props.placeholderAdvanced;
     }
 
     const suggestionList = this.getSuggestionList();
-    const inputComponent = this.props.autoCompleteUri ?
-      (
-        <AutoCompleteInput
-          uri={`${this.props.baseUri}${this.props.autoCompleteUri}`}
-          updateValue={this.updateQuery}
-          placeholder={placeholder || ''}
-          value={query}
-          className={inputClass}
-        />
-      ) : (
-        <input
-          type="search"
-          className={inputClass}
-          placeholder={placeholder}
-          onChange={this.queryChanged}
-          onKeyDown={this.doKeyPress}
-          value={query}
-        />
-      );
+    const inputComponent = this.props.autoCompleteUri ? (
```

Comment:
This isn't part of your changes, but `inputComponent` should be moved out of the render function into its own render function. Same with `shareSearch`. Instead of reference the const you would call the render function directly.